sql-statements/sql-statement-drop-resource-group.md

```diff
@@ -0,0 +1,59 @@
+---
+title: DROP RESOURCE GROUP
+summary: TiDB 数据库中 DROP RESOURCE GROUP 的使用概况。
+---
+
+# DROP RESOURCE GROUP
+
+`DROP RESOURCE GROUP` 语句用于在当前所选数据库中删除资源组。
+
+## 语法图
+
+```ebnf+diagram
+DropResourceGroupStmt:
+    "DROP" "RESOURCE" "GROUP" IfExists ResourceGroupName
+
+IfExists ::=
+    ('IF' 'EXISTS')?
+
+ResourceGroupName:
+    Identifier
+```
+
+> **注意：**
+>
+> `DROP RESOURCE GROUP` 语句只能在全局变量 [`tidb_enable_resource_control`](/system-variables.md#tidb_enable_resource_control-从-v660-版本开始引入) 参数设置为 `ON` 时才能执行。
+
+## 示例
+
+删除名为 `rg1` 的资源组：
+
+```sql
+mysql> DROP RESOURCE GROUP IF EXISTS rg1;
+Query OK, 0 rows affected (0.22 sec)
+mysql> CREATE RESOURCE GROUP IF NOT EXISTS rg1 RU_PER_SEC = 500 BURSTABLE;
+Query OK, 0 rows affected (0.08 sec)
+mysql> SELECT * FROM information_schema.resource_groups WHERE NAME ='rg1';
++------+------------+-----------+
+| NAME | RU_PER_SEC | BURSTABLE |
++------+------------+-----------+
+| rg1  |        500 | YES       |
++------+------------+-----------+-----------+
+1 row in set (0.01 sec)
+
+mysql> DROP RESOURCE GROUP IF EXISTS rg1 ;
+Query OK, 1 rows affected (0.09 sec)
+
+mysql>  SELECT * FROM information_schema.resource_groups WHERE NAME ='rg1';
```

Comment:
```suggestion
mysql> SELECT * FROM information_schema.resource_groups WHERE NAME ='rg1';
```

Reply:
@glorv Please also sync changes to the English PR https://github.com/pingcap/docs/pull/12353. Thanks

Reply:
done.